pages/ksphere/kommander/1.1/upgrade/index.md

```diff
@@ -7,5 +7,17 @@ excerpt: Upgrade Kommander
 enterprise: false
 ---
 
-Kommander is an addon running on a Konvoy cluster, it will be updated in the same way every [Konvoy addon is upgraded](/ksphere/konvoy/latest/upgrade/upgrade-kubernetes-addons/#prepare-for-addons-upgrade).
-When upgrading Kommander, it is recommended to use the equivalent [new version of konvoy CLI and update](/ksphere/konvoy/latest/upgrade/upgrade-cli/) your `cluster.yaml` file according to the instructions.
+Kommander is an addon running on a Konvoy cluster, it will be updated in the same way every [Konvoy addon is upgraded][addons-upgrade].
+
+When upgrading Kommander, it is recommended to use the equivalent [new version of konvoy CLI and update][konvoy-upgrade] your `cluster.yaml` file according to the instructions.
+
+## Kommander cluster creation before Konvoy v1.4.3
+
+In versions before Konvoy v1.4.3 (where Kommander version is v1.0), new Kommander clusters cannot be created. It is recommended to upgrade to the latest version of Konvoy (v1.5.0), which includes Kommander 1.1.0 to be able to create new Kommander clusters.
```

Comment:
I think this is better suited for a "known issues" section in the release notes.

Reply:
What are your thoughts on that, @samvantran ? 

In a way, I agree, because it is a known issue, but on the other hand ... it is for anything with Kommander and only affects upgrades. I suppose we could also put something on upgrades and say "click here to see the known issues with upgrading" - and then link to release notes?

Reply:
I agree with Casie. We do this with Konvoy https://docs.d2iq.com/ksphere/konvoy/1.5/release-notes/#known-issues-and-limitations so it makes sense to put those limitations in the versions that are affected.

Reply:
Thanks for the feedback @cmastr @samvantran 
Commit coming up.